includes/phing/props/drone.props

```diff
@@ -9,3 +9,5 @@ behat.wd_host.url = http://selenium:4444/wd/hub
 share.path = /cache
 project.db.file = ./dump.sql
 project.url.base = http://web:8080
+db.dl.username =
+db.dl.password =
```

Comment:
Why are you doing a reset to values here?

Reply:
Is just to make sure that the variables are empty, to use the values from the ENV vars.
But if it is not needed I can remove.